config/sys.config

```diff
@@ -32,10 +32,10 @@
         {block_sync_batch_limit, 100},
         {honor_quick_sync, true},
         {quick_sync_mode, blessed_snapshot},
-        {blessed_snapshot_block_height, 1160641},
+        {blessed_snapshot_block_height, 1215975},
```

Comment:
If this was a manual snap, I wouldn't. update it here, since no-one on the network will actually have a manually generated snapshot stashed away